google/cloud/forseti/common/gcp_type/firewall_rule.py

```diff
@@ -534,16 +539,20 @@ def __gt__(self, other):
           bool: comparison result
         """
         LOGGER.debug('Checking %s > %s', self, other)
-        return ((self.direction is None or
-                 other.direction is None or
-                 self.direction == other.direction) and
-                (self.network is None or other.network is None or
-                 self.network == other.network) and
-                set(other.source_tags).issubset(self.source_tags) and
-                set(other.target_tags).issubset(self.target_tags) and
-                self.firewall_action > other.firewall_action and
-                ips_in_list(other.source_ranges, self.source_ranges) and
-                ips_in_list(other.destination_ranges, self.destination_ranges))
+        direction = (self.direction is None or
+                     other.direction is None or
+                     self.direction == other.direction)
+        network = (self.network is None or other.network is None or
```

Comment:
Ditto, can you please organize them on a line?
```
network = (self.network is None or
           other.network is None or
           self.network == other.network)
```

Reply:
Done